corehq/apps/receiverwrapper/util.py

```diff
@@ -143,11 +143,13 @@ def get_commcare_version_from_appversion_text(appversion_text):
     '2.4.1'
     >>> get_commcare_version_from_appversion_text(u'संस्करण "2.27.8" (414593)')
     '2.27.8'
+    >>> get_commcare_version_from_appversion_text(u'CommCare Android, आवृत्ती" 2.44.5"(452680). ॲप वि.29635 कॉमर्स आवृत्ती2.44. बिल्ड452680, रोजी तयार केले:2019-01-17')
```

Comment:
I also notice that this will break the `get_version_from_appversion_text` function. Not sure if that's important to fix as well

Reply:
True. We could add for this language, but it might be different for different languages.

@ctsims Do we have any control in formatting this text on mobile end?